packages/plugin-schema-validator/lib/index.ts

```diff
@@ -65,29 +63,19 @@ export class SchemaValidatorPlugin extends Plugin {
       };
     }
 
-    // Compile validator for this event if needed
-    const validators = this.validators!;
-    if (!validators[schemaKey]) {
-      validators[schemaKey] = this.ajv!.compile(this.schemas[schemaKey]);
-    }
-
-    const validator = validators[schemaKey]!;
-    if (event.properties && !(validator(event.properties) === true)) {
-      const errorMessage = validator.errors
-        ? validator.errors.map((e: any) => {
-          let extra = '';
-          if (e.keyword === 'additionalProperties') {
-            extra = ` (${e.params.additionalProperty})`;
-          }
-          return `\`properties${e.dataPath}\` ${e.message}${extra}.`;
-        }).join(' ')
-        : 'An unknown error occurred during validation.';
+    if (event.properties) {
```

Comment:
I feel like this will incorrectly approve all Events without properties as long as it has any schema. Can you please add a test to verify:
 
```
  test('Event with complex schema but empty properties should give validation error ', () => {
    const plugin = new SchemaValidatorPlugin(testSchemas);
    plugin.load();

    const validationResponse = plugin.validate({
      name: 'Event With All Properties',
    });

    expect(validationResponse).toEqual({
      valid: false,
      pluginId: 'schema-validator',
    });
  });
```

Reply:
Added. Logic is fixed.